extensions/blocks/gif/edit.js

```diff
@@ -136,11 +136,11 @@ class GifEdit extends Component {
 				onSubmit={ this.onFormSubmit }
 				ref={ this.textControlRef }
 			>
-				<TextControl
-					className="wp-block-jetpack-gif_input"
-					label={ INPUT_PROMPT }
-					placeholder={ INPUT_PROMPT }
-					onChange={ value => setAttributes( { searchText: value } ) }
+				<input
```

Comment:
Was there an issue with the TextControl component? Is there anything there that can be fixed in core Gutenberg?

Reply:
I'm not sure. The embed block in core doesn't use it, and we are just following that example. Probably worth asking more about that though...

Reply:
I asked here but didn't get any insight! https://wordpress.slack.com/archives/C02QB2JS7/p1581590282345300